Gazebo/models/wsc_aircraft/model.sdf

```diff
@@ -0,0 +1,1803 @@
+        <dynamics>
+          <damping>0.1</damping>
+          <friction>0.1</friction>
+        </dynamics>
+      </axis>      
+    </joint>
+
+    <joint name="front_wheel2_joint" type="revolute">
+      <pose relative_to='front_wheel2'/>
+      <parent>landing_gear</parent>
+      <child>front_wheel2</child>
+      <axis>
+        <xyz>0 0 1</xyz>
+        <dynamics>
+          <damping>0.1</damping>
+          <friction>0.1</friction>
+        </dynamics>
+      </axis>      
+    </joint>
+
+    <joint name="rear_wheel_joint" type="revolute">
+      <pose relative_to='rear_wheel'/>
+      <parent>steering_link</parent>
+      <child>rear_wheel</child>
+      <axis>
+        <xyz>0 0 1</xyz>
+        <dynamics>
+          <damping>15</damping>
+          <friction>1</friction>
+        </dynamics>
+      </axis>      
+    </joint>
+
+    <joint name="steering_wheel_joint" type="revolute">
+      <pose>0 1.1 0.335 0 0 0</pose>
+      <parent>landing_gear</parent>
+      <child>steering_link</child>
+      <axis>
+        <xyz>0 0 1</xyz>
+        <limit>
+          <lower>-0.7</lower>
+          <upper>0.7</upper>
+          <effort>1000</effort>
+          <velocity>100</velocity>
+        </limit>
+        <dynamics>
+          <damping>50</damping>
+          <friction>5</friction>
+        </dynamics>
+      </axis>      
+    </joint>
+
+    <joint name="fuselage_yaw_joint" type="revolute">
+      <pose>0 -1.027 1.9 0 0 0</pose>
+      <parent>wing_body</parent>
+      <child>hang_block_yaw</child>    
+      <axis>
+        <xyz>0 0 1</xyz>
+        <limit>
+          <effort>1e80</effort>
+          <velocity>0.5</velocity>
+          <lower>-0.0</lower>
+          <upper>0.0</upper>
+        </limit>
+        <dynamics>
+          <damping>2e1</damping>
+          <friction>1</friction>
+        </dynamics>
+      </axis>  
+    </joint>
+    
+    <joint name="fuselage_roll_joint" type="revolute">
+      <pose>0 -1.027 1.9 0 0 0</pose>
+      <parent>hang_block_yaw</parent>
+      <child>hang_block_roll</child>    
+      <axis>
+        <xyz>0 1 0</xyz>
+        <limit>
+          <effort>1e80</effort>
+          <velocity>0.5</velocity>
+          <lower>-0.35</lower>
+          <upper>0.35</upper>
+        </limit>
+        <dynamics>
+          <damping>1e2</damping>
+          <friction>1</friction>
+        </dynamics>
+      </axis>  
+    </joint>
+
+    <joint name="fuselage_pitch_joint" type="revolute">
+      <pose>0 -1.027 1.9 0 0 0</pose>
+      <parent>hang_block_roll</parent>
+      <child>hang_block_pitch</child>
+      <axis>
+        <xyz>1 0 0</xyz>
+        <limit>
+          <effort>1e80</effort>
+          <velocity>0.5</velocity>
+          <lower>-0.25</lower>
+          <upper>0.25</upper>
+        </limit>
+        <dynamics>
+          <damping>2e4</damping>
+          <friction>1</friction>
+        </dynamics>
+      </axis>      
+    </joint>
+
+    <joint name="hang_block_joint" type="fixed">

```

Comment:
Depending on the physics solver, you may find instabilities with fixed joints. If this is the case replacing fixed joints with revolute joints with limits constrained to [0, 0] can yield better results. Was the case the Gazebo 11 ODE solver - DART may be better. This is why in the ardupilot_gazebo examples a revolute joint is used for the IMU link.